pl_examples/full_examples/semantic_segmentation/semseg.py

```diff
@@ -0,0 +1,149 @@
+import os
+from argparse import ArgumentParser
+from collections import OrderedDict
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader, Dataset
+from torchvision.models.segmentation import fcn_resnet50
+
+from PIL import Image
+import pytorch_lightning as pl
+
+
+class KITTI(Dataset):
+    def __init__(self, root_path, split='test', img_size=(1242, 376), transform=None):
```

Comment:
pls add simple docstring what is this class about, in particular, explanation about the void and valid labels
personally I would pass the label indexes as a parameter in init with your listed indexes as default... this may simplify re-usage :]

Reply:
I have added these things in the latest commit.